### PR TITLE
Update README; Add support text

### PR DESCRIPTION
**Pull request by techgique**
Simplify the readme and link to wiki pages for more information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,32 @@
...
-NDNP is actually a partnership between the Library of Congress, the
-[National Endowment for the Humanities (NEH)](http://www.neh.gov), and
-cultural heritage organizations
-([awardees](http://chroniclingamerica.loc.gov/awardees/)) across the
-United States who have applied for grants to help digitize newspapers
-in their state. Awardees digitize newspaper microfilm according
-to a set of [specifications](http://www.loc.gov/ndnp/guidelines/)
-and then ship the data back to the Library of Congress where it is
-loaded into Chronicling America.
+Read the [[About Open ONI]] page for more information about the projects origins.
```
Comment (techgique) [inline README.md]: `s/projects/project's/`

File: README.md [hunk 2]
```diff
@@ -1,70 +1,32 @@
...
---------
+We ask that bugs and feature requests be submitted to [Open ONI on GitHub](https://github.com/open-oni/open-oni/issues). Please confine other questions and conversation about Open ONI development and use to [Open ONI's Slack](http://bit.ly/openoni-slack-signup). These guidelines are intended both to keep conversation in the open where it may benefit all of the Open ONI community and to protect developers' already limited time.
 
-Open-ONI releases can be found on our [releases](https://github.com/open-oni/open-oni/releases) page. Open-ONI is alpha software and documentation is incomplete at this time.
+## Install
+Production install instructions are still a work in progress. To evaluate the software, we recommend installing from the latest [release](https://github.com/open-oni/open-oni/releases), rather than checking out the latest changes on the `master` branch.
 
-Install
--------
+If you are interested in development, please read our [Contribute](https:/; Links to wiki/github.com/open-oni/open-oni/wiki/Contribute) page for more details.
```
Comment (techgique) [inline README.md]: Something funny here...

File: README.md [hunk 3]
```diff
@@ -1,70 +1,32 @@
...
 
-**Please note**: for development, look at the [docker setup](docker/README.md).
+## Documentation
+Documentation is in progress in the [wiki](https://github.com/open-oni/open-oni/wiki).## Contact Us
```
Comment (techgique) [inline README.md]: And here...

Copy/paste goof perhaps

**Comment by jechols** [comment]
Can you fry the carriage-return symbols?  In Firefox, they show up as a really weird, distracting, extra-highlighted feature next to the links:

Capture

**Comment by techgique** [comment]
They are there to indicate links that navigate away from the current website. Their display is system and font-dependent.

Here is how they display for me:
f28-firefox-dev-edition

I think using HTML like `<abbr title="External link symbol">⤴</abbr>` will make it more accessible.

Do you think that's adequate?

File: README.md [hunk 4]
```diff
@@ -1,70 +1,73 @@
...
+GitHub](https://github.com/open-oni/open-oni/issues). Please confine other
+questions and conversation about Open ONI development and use to [Open ONI's
+Slack](http://bit.ly/openoni-slack-signup). These guidelines are intended both
+to keep conversation in the open where it may benefit all of the Open ONI
+community and to protect developers' already limited time.
+
+## Contact Us
+We prefer to communicate on the [Open ONI
+Slack](http://bit.ly/openoni-slack-signup), but if you have trouble or prefer
+not to use Slack, email Karin Dalziel (kdalziel2 [at] unl.edu).
```
Comment (karindalziel) [inline README.md]: @techgique should we change this to you? (I don't mind being the contact person still if you don't want to)

Comment (techgique) [reply README.md]: We'll revisit this soon after some more community organization planning decisions are made :)

**Pull request merged**